src/package.json

```
@@ -121,7 +121,10 @@
   "license": "MIT",
   "homepage": "https://frog.fm",
   "repository": "wevm/frog",
-  "authors": ["awkweb.eth", "jxom.eth"],
+  "authors": [
+    "awkweb.eth",
+    "jxom.eth"
+  ],
```

i don't know yet why it keeps changing those
```suggestion
  "authors": ["awkweb.eth", "jxom.eth"],
```